mllib/src/test/scala/org/apache/spark/ml/feature/CountVectorizerSuite.scala

```diff
@@ -115,6 +115,27 @@ class CountVectorizerSuite extends SparkFunSuite with MLlibTestSparkContext
     }
   }
 
+  test("CountVectorizer with binary") {
```

Comment:
Shall I change the name of L192 to "CountVectorizerModel and CountVectorizer with binary"?
In addition, my new test uses a slightly different df for testing CountVectorizer. Shall I use the same df for both cases or I can use different one.

Thanks for your review!

Miao